scripts/upgrades/gen_jovian_upgrade_tx_specs.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -e
```

Comment:
This file is consumed by the script in development here #673. We don't have to commit this file now, but in general once that script is merged we will want to commit files like this along with the autogenerated markdown.